packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -26,7 +25,12 @@ class ScriptLocation {
   }
 
   @override
-  int get hashCode => hashValues(scriptRef, location);
+  int get hashCode {
+    int hash = 17;
+    hash = 0x1fffffff & (hash * 31 + scriptRef.hashCode);
```

Comment:
You might as well use the flutter `hashValues` helper method for consistency.

Reply:
I was trying to avoid a package:flutter import so that the tests for this could run on the dart cli. Perhaps I should add the import and assume that the test will run as a `flutter test`?

Reply:
converted back to using `hashValues()`.